app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java

```diff
@@ -98,11 +97,13 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                         return true;
                     }
                 });
-        // Try pulling data from GraphQL, if not fallback to json from cornell dining
-        NetworkUtilities.getEateries(this);
-        if (JSON_FALLBACK) {
-            // TODO Get Json working
-        }
+
+        getSupportFragmentManager().beginTransaction().replace(R.id.frame_fragment_holder,
+                mainListFragment).commit();
+
+        // Try pulling data from GraphQL
+        NetworkUtilities.getEateries(this, mainListFragment);
+        NetworkUtilities.getCtEateries(this);
```

Comment:
regardless if we are pulling from cache or not, we're making networking calls?

Reply:
Yeah - the current data flow is we try and populate the models with the cache, and then on each app launch we make new networking calls and with a positive response we update the cache and models on the device. This doesn't increase the number of networking calls from before and I agree it is redundant but I don't want to throw in a system where we only make updating calls once every time period yet. Lmk if you think otherwise and we can fix this up.

Reply:
discussed offline. 

it's your call: we can either remove the redundancy in this pr, or you can make a git issue and resolve it in your next pr. if you choose the latter, make sure you reply with the issue # so that we can track the background info on the issue.

Reply:
Will be targeted next with issue #161